examples/hello-world-django/README.md

```diff
@@ -0,0 +1,49 @@
+# hello-world
+
+This simple Django app returns "Hello World". This app uses the `configure_opentelemetry` method from `honeycomb.opentelemetry` to set up OpenTelemetry to export data to Honeycomb through configuration options set in the app's code. It is also possible to set configuration options through the `opentelemetry_instrument` command (see the instructions for automatic instrumenetation [here](https://docs.honeycomb.io/getting-data-in/opentelemetry/python-distro/#automatic-instrumentation)).
+
+## Prerequisites
+
+You'll need [Poetry](https://python-poetry.org/) installed to run the example. Poetry automatically creates a virtual environment to run the example in so you don't need to manage one yourself.
+
+## Running the example
+
+Install application dependencies:
+
+```bash
+poetry install
+```
+
+Run the application:
+
+```bash
+poetry run python manage.py runserver
+```
+
+Then navigate to http://127.0.0.1:8000 as shown in the command output and you should see `Hello, world`.
+
+## Distro Instrumentation Example
+
+This app uses configuration configures the OpenTelemetry SDK programmatically in [manage.py](./manage.py).
+Alternitively, you can use environment variables as parameters like below:
+
+```python
+configure_opentelemetry(
+    HoneycombOptions(
+        debug=True,
+        apikey=os.getenv("HONEYCOMB_API_KEY"),
+        service_name="hello-world-django"
+    )
+)
+```
+
+This app will send traces to local console with the configured `debug=True`.
+
+To send to Honeycomb, set your API Key:
+
+```bash
+HONEYCOMB_API_KEY="your-api-key" poetry run python3 app.py
```

Comment:
Update this example command to use the same as earlier in this README for the django app.
```suggestion
HONEYCOMB_API_KEY="your-api-key" poetry run python manage.py runserver
```